packages/client/src/Subscription.ts

```diff
@@ -2,12 +2,12 @@
  * The client.subscribe() return value.
  * Primary interface for consuming StreamMessages.
  */
-import { SPID, SPIDKeyShape } from 'streamr-client-protocol'
+import { SPID, SPIDKeyShape, StreamID } from 'streamr-client-protocol'
 import MessageStream, { MessageStreamOptions, MessageStreamOnMessage } from './MessageStream'
 import SubscriptionSession from './SubscriptionSession'
 
 export type SubscriptionOptions = {
```

Comment:
Could remove this type if it is not used anywhere

Reply:
removed